src/app/app-routing.module.ts

```diff
@@ -14,7 +15,8 @@ const routes: Routes = [
     { path: 'news', component: NewsComponent },
     { path: 'team', component: TeamComponent },
     { path: 'contact-us', component: ContactUsComponent },
-    { path: '**', redirectTo: '/', pathMatch: 'full' }
+    { path: 'main', component: MainComponent },
```

Comment:
Setting path to `''` over `'main'` will result in the main page showing up at `calgarysolarcar.ca` instead of `calgarysolarcar.ca/main`

It's nitpicking a bit, but IMO just makes everything feel a bit cleaner.